examples/label_studio_annotation/README.md

```diff
@@ -0,0 +1,318 @@
+
+```shell
+cd cloud_config/aws
+
+aws s3api put-bucket-cors --bucket $S3_BUCKET_NAME --cors-configuration file://cors.json
+
+cd ../..
+```
+
+Now you can get to the fun part of setting up Label Studio and working with ZenML:
+
+```shell
+zenml stack copy default <YOUR_AWS_ZENML_STACK_NAME>
+
+zenml stack set <YOUR_AWS_ZENML_STACK_NAME>
+
+zenml secrets-manager register <YOUR_SECRETS_MANAGER> --flavor=aws --region_name=<AWS_REGION_NAME>
+
+zenml stack update <YOUR_AWS_ZENML_STACK_NAME> -x <YOUR_SECRETS_MANAGER>
+
+# copy down the credentials output from this command
+aws sts get-session-token --profile=<YOUR_AWS_CONFIG_PROFILE_NAME>
+```
+
+Use the credentials output from the command to get your `<LABEL_STUDIO_ACCESS_KEY_ID>`, `<LABEL_STUDIO_SECRET_ACCESS_KEY>`, and `<LABEL_STUDIO_AWS_SESSION_TOKEN>` which we will use to register a secret.
+
+```shell
+# this secret must be named 'aws_label_studio'
+zenml secret register aws_label_studio --schema=aws_label_studio --aws_access_key_id="<LABEL_STUDIO_ACCESS_KEY_ID>" --aws_secret_access_key="<LABEL_STUDIO_SECRET_ACCESS_KEY>" --aws_session_token="<LABEL_STUDIO_AWS_SESSION_TOKEN>"
+
+# use your standard access key id and secret access key from ~/.aws/credentials here
+zenml secret register <YOUR_AWS_SECRET_NAME> --schema=aws_label_studio--aws_access_key_id="<YOUR_ACCESS_KEY_ID>" --aws_secret_access_key="<YOUR_SECRET_ACCESS_KEY>"
+
+zenml artifact-store register <YOUR_CLOUD_ARTIFACT_STORE> --flavor=s3 --path=s3://<YOUR_S3_BUCKET_NAME> --authentication_secret="<YOUR_AWS_SECRET_NAME>"
+
+zenml stack update <YOUR_AWS_ZENML_STACK_NAME> -a <YOUR_CLOUD_ARTIFACT_STORE>
+```
+
+You will next need to obtain your Label Studio API key. This will give you access to the web annotation interface.
+
+```shell
+# start a temporary / one-off label-studio instance to get your API key
+label-studio start -p 8094
+```
+
+First order of business is to sign up if you haven't done so already, using an email and a password. Then visit http://localhost:8094/ to log in, and then visit http://localhost:8094/user/account and get your Label Studio API key (from the upper right hand corner). You will need it for the next step. Ctrl-c out of the Label Studio server that is running on the terminal.
+
+```shell
+zenml secret register <LABEL_STUDIO_SECRET_NAME> --api_key="<YOUR_API_KEY>"
+
+zenml annotator register <YOUR_LABEL_STUDIO_ANNOTATOR> --flavor label_studio --authentication_secret="<LABEL_STUDIO_SECRET_NAME>"
+
+zenml stack update <YOUR_AWS_ZENML_STACK_NAME> -an <YOUR_LABEL_STUDIO_ANNOTATOR>
+
+zenml stack up
+```
+
+### ▶️ Run the Code
+
+
+This will initialize the daemon server which Label Studio requires, albeit running on a default port of 8093.
+
+Run the pipeline with:
+
+```shell
+python run.py aws --train
+python run.py aws --inference
+# Now do some annotation using Label studio
+python run.py aws --train --rerun
+python run.py aws --inference --rerun
+```
+
+## CLI Commands for the Label Studio Integration
```

Comment:
Instead of listing what commands there are I'd recommend to show it at the respective place in the previous section, e.g., if you follow the structure I proposed in the previous comment then you could under 2. do `zenml annotator dataset list` at the end to show that there is now something in label studio, and then under 3. have the `zenml annotator dataset ... stats` command to show that X images need to be labeled now and then run `zenml annotator dataset ... annotate` to open the UI and do the annotation. Screenshots of these steps (label studio UI, stats CLI output, ...) would also be nice to include here so users can understand the example without having to actually run it.